test/container/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine
+
+RUN apk update && apk add --no-cache curl bash util-linux grep
+
+CMD ["bash", "-c", "while true; do sleep 120; done"]
```

Comment:
`sleep infinity` is also valid

Reply:
Sadly only supported by gnu sleep (so not on alpine or mac even)